lib/matplotlib/ticker.py

```diff
@@ -1872,7 +1914,13 @@ def tick_values(self, vmin, vmax):
         return self.raise_if_exceeds(np.array(ticklocs))
 
 
-class AutoLocator(MaxNLocator):
```

Comment:
I am talking my self out of any argument I have against doing this, but still don't like it.